rdr_service/model/code.py

```diff
@@ -49,9 +49,12 @@ class _CodeBase(object):
     @rdr_dictionary_show_unique_values
     """
     value = Column("value", String(80), nullable=False)
```

Comment:
This isn't a big issue, but is a limit of 80 characters a safe-enough buffer for longer codes?

Reply:
I believe so. I'm not entirely sure about specs in other systems, but we haven't hit codes that long yet and we've rarely gone over 50 chars. I imagine we can handle that scenario and investigation later if we need to.

Reply:
Sounds good.